models/news.model.js

```diff
@@ -9,21 +9,41 @@ const readTopics = () => {
     });
 };
 
-const readArticles = () => {
+const readArticles = ((req, sortBy = "created_at", order = "DESC")=>{
+  const { topic } = req.query;
+  const acceptedSorBy = ["created_at", "article_id", "title", "topic", "votes" ];
+  const acceptedOrderBy = ["DESC", "ASC"];
+ 
+  if (!acceptedSorBy.includes(sortBy) || !acceptedOrderBy.includes(order)){
+  return Promise.reject({status: 400, msg: 'Please sort and oredr by acceptable parameters'})
+} else 
+if (!topic){ 
   let countString =  `SELECT articles.article_id, articles.title, articles.topic, 
```

Comment:
This should be a const unless you are changing it at somepoint. I have a feeling that you might need to change it at some point based on the additional tests we have asked you to write.